go.mod

```diff
@@ -4,7 +4,7 @@ go 1.18
 
 // See the following link for reasoning on why we need the replacement:
 // https://discuss.dgraph.io/t/error-mremap-size-mismatch-on-arm64/15333/8
-replace github.com/dgraph-io/ristretto v0.1.0 => github.com/46bit/ristretto v0.1.0-with-arm-fix
+//replace github.com/dgraph-io/ristretto v0.1.0 => github.com/46bit/ristretto v0.1.0-with-arm-fix
```

Comment:
I think this is going to cause issues for everyone on `ARM64`. Does it not work for you at all?

Reply:
Well I think including it will cause issues for everyone. The repo has been deleted: https://github.com/46bit/ristretto

I commented it out when I should probably remove it all together

Reply:
If the repo no longer exists, let's delete this comment altogether